src/resources/common.yml

```diff
@@ -470,3 +470,9 @@ trackMixer:
 caption:
   en: Caption
   de: Legende
+defaultVolumePreset:
+  en: Default
+  de: Standard
+volumesPreset:
```

Comment:
In all other places there is no "s" at the end of "volume". Let's also drop it here?

Reply:
#dropped